components/TaggedValue/TaggedValue.jsx

```diff
@@ -0,0 +1,30 @@
+require('./TaggedValue.scss')
+
+import React from 'react'
+import classNames from 'classnames'
+
+const TaggedValue = ({title,subText,style,count}) => {  
+  const taggedValueClass = classNames(
+    'TaggedValue',
+    { [`${style}`]: style,
```

Comment:
Benefit is that we have relatively simple code `[style] : true` instead of  `[``${style}``]: style` and it would be more like themes to use class for default style as well. `tagged-value-bg` class would contain rules that are rare to change for the component, mostly we would be changing background and font colors which we can move to class. This way we can support more themes in the component itself, the caller may not need to define the background and font colors. They can simply specify the supported theme name.